tests/Feature/PreviewDomainPortOverridesTest.php

```
@@ -0,0 +1,604 @@
+            'frontend' => ['domain' => 'https://one.example.com:3000,https://two.example.com:8080'],
+        ]),
+    ]);
+
+    $preview = createPreviewForPortTests($this->application, 124);
+
+    parseDockerComposeFile($this->application->fresh(), pull_request_id: 124, preview_id: $preview->id);
+
+    $preview->refresh();
+
+    expect(explode(',', (string) $preview->fqdn))->toHaveCount(2)
+        ->and($preview->domain_port_overrides)->toHaveCount(2)
```

PHPStan (2.2.8)

[warning] Unable to resolve the template type TAndValue in call to method Pest\Expectation<list<string>|null>::and()
See: https://phpstan.org/blog/solving-phpstan-error-unable-to-resolve-template-type

(argument.templateType)